packages/builders/src/messages/Message.ts

```diff
@@ -56,7 +56,9 @@ export interface MessageBuilderData
 /**
  * A builder that creates API-compatible JSON data for messages.
  */
-export class MessageBuilder implements JSONEncodable<RESTPostAPIChannelMessageJSONBody> {
+export class MessageBuilder
+	implements JSONEncodable<RESTPostAPIChannelMessageJSONBody>, FileBodyEncodeable<RESTPostAPIChannelMessageJSONBody>
```

Comment:
```suggestion
	implements FileBodyEncodeable<RESTPostAPIChannelMessageJSONBody>
```

This is redundant considering `FileBodyEncodeable` extends `JSONEncodable`

Reply:
I'm aware, but I'm honestly for leaving it in just for extra readability without knowing what a FileBodyEncodeable is (arguably beneficial for docs too?). Thoughts @vladfrangu

Reply:
For docs it would (should) show the toJSON() on the extended interface already, so not sure of the benefit